tests/test_irbempy.py

```diff
@@ -347,6 +475,19 @@ def test_coord_trans(self):
             [ 1.91462214,  0.06992421,  0.57387514]])
         numpy.testing.assert_almost_equal(expected, ib.coord_trans(self.loci, 'GSM', 'car'))
 
+    def test_coord_trans_swap(self):
+        """test coord_trans with invalid sysaxesout requiring a carsph swap"""
+        # re-assign ticks so this test can be run in isolation
+        self.loci.ticks = self.ticks
```

Comment:
I'm not sure what this means. It's assigning ticks to the loci, which didn't have it before, but how is that isolating? (The setUp is run for every test, so one test shouldn't affect another)

Reply:
Somehow, I was not aware of that last part. I'll remove that comment.